### ISSUE TITLE
pylint suppression comments trigger new lint

### ISSUE THREAD

**Issue by jml** [twisted/twistedchecker#103]
I have the line 

```python
    def global_tcpip_forward(self, data):  # pylint: disable=C0103
```

Which causes the following warning to be emitted:

```
W9402: 88,0: The first letter of comment should be capitalized
```

This is silly. I would suggest dropping the check altogether.

**Comment by wsanchez** [comment]
I agree.  This check provides no real benefit.  Even when we're dealing with actual words, are the docs really so much more readable when one writes:

```
foo = 4  # A counter
```

versus:

```
foo = 4  # a counter
```

Is it really worth out time to be chasing this nonsense down?  And I'm often starting a comment with the name of a symbol, so to work around this, I need to rephrase my comment.
